tools/run-nrtest.cmd

```diff
@@ -7,16 +7,21 @@
 ::          US EPA - ORD/NRMRL
 ::
 ::  Arguments:
-::    1 - version/build identifier
+::    1 - (SUT build identifier)
 ::    2 - (test suite path)
 ::
 
 @echo off
 setlocal
 
+set "REF_BUILD_ID=212_1"
```

Comment:
@michaeltryby, I'm sorry for my naïvety.. what exactly does this correspond to? I believe it is an appveyor number

Reply:
%APPVEYOR_BUILD_NUMBER%_%APPVEYOR_JOB_NUMBER%

Reply:
This has been eliminated. REF_BUILD_ID is now determined automatically.

Reply:
I wasn't done with the PR before. Now I am.